src/main/java/com/minecolonies/commands/ISubCommand.java

```diff
@@ -0,0 +1,57 @@
+package com.minecolonies.commands;
+
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.math.BlockPos;
+import org.jetbrains.annotations.NotNull;
+
+import javax.annotation.Nullable;
+import java.util.List;
+
+/**
+ * Override this to specify a sub command.
+ */
+public interface ISubCommand
+{
+
+    /**
+     * Gets the usage string for the command.
+     *
+     * @param sender the CommandSender who gets to see the usage string
+     * @return this subcommands part of the usage string.
+     */
+    @NotNull
+    String getCommandUsage(@NotNull final ICommandSender sender);
+
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)